src/pages/ClaimReward.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * This program has been developed by students from the bachelor Computer Science at Utrecht University within the Software Project course.
+ * © Copyright Utrecht University (Department of Information and Computing Sciences)
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export type ClaimRewardData = {
+  division: number;
+};
+
+import { Controller, useForm, useWatch } from 'react-hook-form';
+import { Label } from '../components/ui/Label';
+import { Slider } from '../components/ui/Slider';
+import { Button } from '../components/ui/Button';
+import TokenAmount from '../components/ui/TokenAmount';
+import { MainCard } from '../components/ui/MainCard';
+import { IconBaseProps } from 'react-icons/lib';
+import { HiOutlineCurrencyDollar } from 'react-icons/hi2';
+import { Card } from '../components/ui/Card';
+import { TOKENS } from '../lib/constants/tokens';
+
+/*
+ * repToBeClaimed is the total amount of reputation the user may claim
+ * repToMonetaryFactor is the amount of monetary token 1 rep may be converted to.
+ * */
+export const ClaimReward = ({
+  repToBeClaimed,
+  repToMonetaryFactor,
+}: {
+  repToBeClaimed: number;
+  repToMonetaryFactor: number;
+}) => {
+  const {
+    register,
+    formState: { errors },
+    control,
+    handleSubmit,
+  } = useForm<ClaimRewardData>({
+    defaultValues: { division: 100 },
+  });
+
+  const name_division = 'division';
+
+  const division = useWatch({
+    control,
+    name: name_division,
+  });
+
+  const reputation = division * repToBeClaimed * 0.01 || 0;
+  const monetary =
+    (100 - division) * repToBeClaimed * repToMonetaryFactor * 0.01 || 0;
+
+  const onSubmit = (data: ClaimRewardData) => {
+    console.log(data);
+  };
+
+  return (
+    <MainCard icon={HiOutlineCurrencyDollar} header="Claim Reward">
+      <form onSubmit={handleSubmit(onSubmit)} className="flex flex-col gap-1">
+        <Label
+          htmlFor="division"
+          tooltip="How to divide the reward between monetary value and repuation"
```

Comment:
Typo in "repuation"

Reply:
Also, I would suggest making the tooltip text the following (assuming you agree with the comment below to rename "division" to "distribution":
```suggestion
          tooltip={`How much of your rewards you wish to receive in ${TOKENS.rep.symbol}, and how much in ${TOKENS.secoin.symbol}`}
```